source/common/websocket/codec.h

```diff
@@ -0,0 +1,178 @@
+#pragma once
+
+#include <array>
+#include <cstdint>
+#include <vector>
+
+#include "envoy/buffer/buffer.h"
+
+namespace Envoy {
+namespace WebSocket {
+
+// Opcodes (https://datatracker.ietf.org/doc/html/rfc6455#section-11.8)
+constexpr uint8_t FRAME_OPCODE_CONTINUATION = 0;
```

Comment:
All these constexpr globals should be named kSomeName as per the style guide:

https://google.github.io/styleguide/cppguide.html#Constant_Names